openlibrary/catalog/add_book/__init__.py

```diff
@@ -951,6 +1001,7 @@ def load(rec, account_key=None):
     need_work_save = need_edition_save = False
     work: dict[str, Any]
     edition: Edition = web.ctx.site.get(match)
```

Comment:
Maybe this should be called existing_edition

Reply:
I have so renamed this.